test/metabase/driver/sql_jdbc/sync/describe_table_test.clj

```diff
@@ -79,6 +92,16 @@
              :json-unfolding             false}}}
          (sql-jdbc.describe-table/describe-table :h2 (mt/id) {:name "VENUES"}))))
 
+(defn- describe-fields-for-table [db table]
+  (let [driver (driver.u/database->driver db)]
+    (sort-by :database-position
+             (if (driver/database-supports? driver :describe-fields db)
+               (vec (sql-jdbc.describe-table/describe-fields driver db
```

Comment:
nit: it seems wasteful to materialize a full vector just then build an array and sort it, for large very N we could optimize this further. Definitely not a bottleneck though.

Reply:
yeah, this is just a test helper for very small N. Not going to do this

Reply:
Somehow I missed this was just for a single table.